resources/views/auth/application/files.blade.php

```diff
@@ -29,49 +25,35 @@
                     </li>
                 </ul>
             </blockquote>
-
-            <div style="margin: 0 20px 0 20px;">
-                @forelse ($user->application->files ?? [] as $file)
-                    @if (!$loop->first)
-                        <div class="divider"></div>
-                    @endif
-                    <div class="row" style="margin-bottom: 0; padding: 10px">
-                        <form method="POST"
-                              action="{{ route('application.store', ['page' => 'files.delete', 'id' => $file->id]) }}"
-                              enctype='multipart/form-data'>
-                            @csrf
-
-                            <div class="col s10" style="margin-top: 5px">
-                                <a href="{{ url($file->path) }}" target="_blank">{{ $file->name }}</a>
-                            </div>
-                            <div class="col s2">
-                                <x-input.button floating class="right btn-small" icon="delete"/>
-                            </div>
-                        </form>
-                    </div>
-                @empty
-                    <p>Még nem töltött fel egy fájlt sem.</p>
-                @endforelse
-            </div>
-        </div>
-    </div>
-    {{-- upload --}}
-    <div class="card">
-        <form method="POST" action="{{ route('application.store', ['page' => 'files']) }}"
-              enctype='multipart/form-data'>
-            @csrf
-            <div class="card-content">
-                <div class="card-title">Feltöltés</div>
+            <form method="POST" action="{{ route('application.store', ['page' => 'files']) }}"
+                  enctype='multipart/form-data'>
+                @csrf
                 <div class="row">
-                    <x-input.file s=12 m=6 id="file" size="2000000" accept=".pdf,.jpg,.png,.jpeg" text="general.browse" required/>
+                    <x-input.file s=12 m=6 id="file" size="2000000" accept=".pdf,.jpg,.png,.jpeg" text="general.browse" helper=".pdf,.jpg,.png,.jpeg fájlok tölthetőek fel, maximum 2 MB-os méretig." required/>
```

Comment:
The helper does not seem to work for me.

Reply:
I updated the file component. you need to reset cache.